### PR TITLE
Add Travis CI support to IoT.js

### PR DESCRIPTION
**Pull request by akiss77**
IoT.js-DCO-1.0-Signed-off-by: Akos Kiss akiss@inf.u-szeged.hu

### CODE REVIEW

**Comment by LaszloLango** [comment]
LGTM

**Comment by akiss77** [comment]
Updated to current master. And travis is on already, so it already checked this :)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+install:
+
+script: "tools/build.py $OPTS"
+
+env:
+  - OPTS="--buildtype=debug"
+  - OPTS="--buildtype=release"
+  - OPTS="--jerry-lto"
+  - OPTS="--no-snapshot"
```
Comment (zherczeg) [inline .travis.yml]: No snapshot?

Comment (akiss77) [inline .travis.yml]: Yes. That's a valid command line argument of `tools/build.py`. It disables the use of the snapshot feature. In this case, it runs a test job to check that variant as well. (Was that what you asked?)

Comment (akiss77) [inline .travis.yml]: Yes. That's a valid command line argument of `tools/build.py`. It disables the use of the snapshot feature. In this case, it runs a test job to check that variant as well. (Was that what you asked?)

Comment (akiss77) [inline .travis.yml]: Yes. That's a valid command line argument of `tools/build.py`. It disables the use of the snapshot feature. In this case, it runs a test job to check that variant as well. (Was that what you asked?)

Comment (zherczeg) [inline .travis.yml]: So it runs  both with and without snapshot? That is ok.

Comment (akiss77) [inline .travis.yml]: Yes.

**Comment by zherczeg** [comment]
LGTM

**Pull request merged**